tests/quantization/torchao/test_torchao.py

```diff
@@ -611,10 +630,12 @@ def _test_quant_type(self, quantization_config, expected_slice):
         pipe = FluxPipeline(**components)
         pipe.enable_model_cpu_offload()
 
+        weight = pipe.transformer.transformer_blocks[0].ff.net[2].weight
+        self.assertTrue(isinstance(weight, (AffineQuantizedTensor, LinearActivationQuantizedTensor)))
```

Comment:
Just to be sure that quantization is actually working. Weight-only quantization would result in `AffineQuantizedTensor`, whereas as Weight+Activation quantization would result in `LinearActivationQuantizedTensor`